src/main/java/io/github/thebusybiscuit/cscorelib2/protection/ProtectionManager.java

```diff
@@ -70,6 +70,7 @@ public ProtectionManager(@NonNull Server server) {
         registerModule(server, "BentoBox", plugin -> new BentoBoxProtectionModule(plugin));
         registerModule(server, "BlockLocker", plugin -> new BlockLockerProtectionModule(plugin));
         registerModule(server, "Lands", plugin -> new LandsProtectionModule(plugin));
+        registerModule(server, "ChestProtect", plugin -> new ChestProtectProtectionModule(plugin));
```

Comment:
I was wondering why the build was marked as a failure. Turns out there is an import missing for this XD

Reply:
Oh well, you got me. Thanks, is solved now.